### PR TITLE
Documentation

### PR DESCRIPTION
**Pull request by paweljw**

### CODE REVIEW

**Comment by paweljw** [comment]
@tkowalewski ready for your review

**Comment by paweljw** [comment]
@tkowalewski GFM support through redcardpet seems to be a bit broken in yard :( worked around it in the last commit

File: lib/ccs/commands/copy.rb [hunk 1]
```diff
@@ -2,7 +2,25 @@
...
+    # and decryption as necessary.
+    #
+    # The target and destinations can be:
+    #
+    # - STDIN/STDOUT: a `-` sign is interpreted as these standard streams
+    #
+    # - The Occson server: strings beginning with `ccs://` or `http(s)://` are interpreted as such
+    #
+    # - local files: everything not matching the previous descriptions is assumed to
+    #   be a path on the local systm
```
Comment (tkowalewski) [inline lib/ccs/commands/copy.rb:15]: system not systm :)

File: lib/ccs/decrypter.rb [hunk 2]
```diff
@@ -1,12 +1,28 @@
...
+  # expected in bytes 8..15, with ciphertext occupying the
+  # further bytes.
   class Decrypter
+    # Constructs a Decrypter instance with given passphrase and content.
+    #
+    # @example
+    #   Ccs::Decrypter.new('the content passphrase', content)
+    #  
+    # @param passphrase [String] Passphrase for content decryption
+    #
```
Comment (tkowalewski) [inline lib/ccs/decrypter.rb:16]: We can remove this line?

**Review by tkowalewski — APPROVED** [review]
Great job!

**Pull request merged**